Application/LoadScreen.py

```diff
@@ -25,9 +24,13 @@ def load_ui(self):
         self.ui = loader.load(ui_file, self)
         ui_file.close()
 
+        # Setup loading GIF
+        gif = QMovie("Images/Logos/LoadingDroneLogo.gif")
+        gif.setSpeed(125)
+        self.ui.label_Logo.setMovie(gif)
+        gif.start()
+
```

Comment:
Can we do the app refresh thing here? The .gif takes a bit to start on my computer.

Reply:
It should start the gif faster now. There will always be a bit of a lag because the gui cannot run the gif when anything else is running on the main thread, but it is starting basically instantly for me now.